package-lock.json

```diff
@@ -7524,7 +7524,6 @@
       "version": "1.1.3",
       "resolved": "https://registry.npmjs.org/define-properties/-/define-properties-1.1.3.tgz",
       "integrity": "sha512-3MqfYKj2lLzdMSf8ZIZE/V+Zuy+BgD6f164e8K2w7dgnpKArBDerGYpM46IYYcjnkdPNMjPk9A6VFB8+3SKlXQ==",
-      "dev": true,
```

Comment:
This is a genuine question: why get rid of "dev" ? It shouldn't have been here in the first place ?

Reply:
This file is auto-generated by `npm` and updates with their policies.